### PR TITLE
Everything: Move to SPDX license identifiers in all files.

### PR DESCRIPTION
**Pull request by bgianfo**
SPDX License Identifiers are a more compact / standardized
way of representing file license information.

See: https://spdx.dev/resources/use/#identifiers

This was done with the `ambr` search and replace tool.

 ambr --no-parent-ignore --key-from-file --rep-from-file key.txt rep.txt *

### CODE REVIEW

File: AK/Checked.h [hunk 1]
```diff
@@ -1,28 +1,8 @@
 /*
  * Copyright (C) 2011-2019 Apple Inc. All rights reserved.
```
Comment (emanuele6) [inline AK/Checked.h:2]: Can this really be replaced?

Comment (bgianfo) [reply AK/Checked.h:2]: No I'll undo that thanks for checking... I found a few of these manually.. but hard to automate.

Comment (bgianfo) [reply AK/Checked.h]: Ok fixed, I manually poked through the diff in vim ... I think this is the only case of this.

Comment (emanuele6) [reply AK/Checked.h]: yep, I just pulled your branch and did a quick `git show | grep -F 'Copyright (c)' | sed -E 's@[0-9-]+,?@@' | sort -u | less` (72 results). everything looks good.

Comment (bgianfo) [reply AK/Checked.h]: Thank you for double checking!

**Pull request merged**